### PR TITLE
VIRAL and NGU tokens

### PR DESCRIPTION
**Pull request by larkben**
Viral - @viraldecay on Discord and Number Go Up - @elvisgan also from Discord.

### CODE REVIEW

File: tokens/mainnet.json
Comment (BURLY7) [inline tokens/mainnet.json:1]: Wen Number Go Up?

**Comment by polarker** [comment]
@larkben Could you rebase on the master branch? There are conflicts.

**Comment by polarker** [comment]
@larkben Test failed. It seems the name onchain is not matched with the metadata in this PR.

**Comment by larkben** [comment]
Thanks Cheng lemme check

**Comment by larkben** [comment]
@polarker looks like its all good now

**Pull request merged**